docker/install.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
 #  Copyright (C) 2018-2023 LEIDOS.
-# 
```

Comment:
Not actually a comment on the `docker/install.sh` file changes, but can you update the PR description to indicate that the `docker/install.sh` file was updated rather than `docker/checkout.bash`?

Reply:
Done